BearStudio/start-ui-native
# Add storybook and Button colors and variants inside

## Aim of this PR
We needed a better way to show components, instead of having a custom screen, because sometime, the components (like layouts) need their own screen.
This is why I added the `@storybook/react-native` package and set it up.

## How to test
 - Run the app
 - Open the developer menu
 - Click on the "Storybook" option
 - You should see the two stories "Button colors" and "Button variants"

## Screenshots
| Dev menu  | Stories  | Button color story  |
|---|---|---|
|![Screenshot_20210820_105653_com startuinativeapp](https://user-images.githubusercontent.com/27200095/130208802-cbe5c75f-f0a2-4e8d-b72c-327f43685ae1.jpg) |![Screenshot_20210820_105636_com startuinativeapp](https://user-images.githubusercontent.com/27200095/130208806-525f789c-8182-44fe-92ce-07a0b66a9037.jpg) | ![Screenshot_20210820_105644_com startuinativeapp](https://user-images.githubusercontent.com/27200095/130208804-23b83161-2d8e-490d-8e95-61fb1c622efd.jpg) |
